helm/values.yaml

```diff
@@ -34,9 +34,9 @@ secret:
 #     See https://github.com/voxel51/fiftyone-teams-app-deploy/tree/main/helm#enabling-fiftyone-teams-plugins
 #   env:
 #     FIFTYONE_PLUGINS_DIR: /opt/plugins
+#   dnsName: your-api.hostname.here
```

Comment:
I think `dnsName` should be at the top (partly because of alphabetization, partly because it's that way in `teamsAppSettings` [nit]